recipes/libjpeg/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import shutil
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+
+class LibjpegConan(ConanFile):
+    name = "libjpeg"
+    version = "9c"
+    description = "Libjpeg is a widely used C library for reading and writing JPEG image files."
+    url = "http://github.com/bincrafters/conan-libjpeg"
+    license = "http://ijg.org/files/README"
+    homepage = "http://ijg.org"
+    exports = ["LICENSE.md"]
```

Comment:
```suggestion
```

Reply:
why? we no longer export licenses for packages?

Reply:
hmm, you are right. But we need export from root level, otherwise we would need to replicate the same file for each recipe.